jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPresenter.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.es.client.editors.jobdetails;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.event.Event;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+import org.jboss.errai.common.client.api.Caller;
+import org.jbpm.workbench.es.client.editors.events.JobSelectedEvent;
+import org.jbpm.workbench.es.client.i18n.Constants;
+import org.jbpm.workbench.es.model.RequestDetails;
+import org.jbpm.workbench.es.model.RequestSummary;
+import org.jbpm.workbench.es.service.ExecutorService;
+import org.uberfire.client.annotations.DefaultPosition;
+import org.uberfire.client.annotations.WorkbenchPartTitle;
+import org.uberfire.client.annotations.WorkbenchPartView;
+import org.uberfire.client.annotations.WorkbenchScreen;
+import org.uberfire.client.mvp.PlaceManager;
+import org.uberfire.client.mvp.UberElement;
+import org.uberfire.client.workbench.events.ChangeTitleWidgetEvent;
+import org.uberfire.ext.widgets.common.client.menu.RefreshMenuBuilder;
+import org.uberfire.lifecycle.OnStartup;
+import org.uberfire.mvp.PlaceRequest;
+import org.uberfire.workbench.model.CompassPosition;
+import org.uberfire.workbench.model.Position;
+
+@Dependent
+@WorkbenchScreen(identifier = JobDetailsPresenter.SCREEN_ID, preferredWidth = 655)
+public class JobDetailsPresenter implements RefreshMenuBuilder.SupportsRefresh {
+
+    public static final String SCREEN_ID = "Job Details";
+
+    @Inject
+    public JobDetailsView view;
+
+    @Inject
+    PlaceManager placeManager;
+
+    @Inject
+    private Event<ChangeTitleWidgetEvent> changeTitleWidgetEvent;
+
+    private PlaceRequest place;
+
+    private String serverTemplateId;
+    private String deploymentId;
+    private Long jobId;
+    private String processName;
```

Comment:
@nmirasch I'm a bit reluctant to include the process related data in the details. Although that is visible on the table, this data is not part of the job domain information. If a job is part of a process, the instance id is shown the parameter table already. @bsremac wdyt?

Reply:
@cristianonicolai @nmirasch Yes, I agree here. I would limit the information exposed to that we are exposing to jobs which don't belong to a process definition.
The approach also adds to information consistency we are offering for jobs.